fpm-alpine/php.ini

```diff
@@ -0,0 +1,4 @@
+always_populate_raw_post_data=-1
+extension=geoip.so
```

Comment:
this is not present in other images, should we add it to others, or remove it from here?